testing/jupyterlab/tests/a11y.test.ts

```diff
@@ -0,0 +1,111 @@
+import { expect } from "@playwright/test";
+import { injectAxe, getViolations, Options } from "axe-playwright";
+import { test } from "@jupyterlab/galata";
+import { createHtmlReport } from "axe-html-reporter";
+import fs from "fs";
+import * as path from "path";
+
+const LORENZ = "Lorenz.ipynb";
```

Comment:
Is there a way to no hardcode this?

Reply:
I'm not sure I understand what you mean

Reply:
Are we always going to test against Lorentz? or might there be instances in which we might need to use other notebooks? in which case we might want to give flexibility to change this instead of having this as a default `const`